README.md

```diff
@@ -1 +1,68 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **wycliff susiku mubita*
+
+- GitHub: [@githubhandle](https://github.com/susiku)
+- Twitter: [@twitterhandle](https://twitter.com/@ws_mubita)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/wycliff-susiku-mubita-010423ba)
+
+👤 
```

Comment:
[OPTIONAL] Kindly remove this icon. as there is only one author in that project.